tests/manage/pv_services/test_pvc_resize.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version(['>4.1', '<4.5'])
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-00000")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-0000")
+        )
+    ]
+)
+def test_pvc_resize(interface, storageclass_factory, pvc_factory, pod_factory):
+    """
+    Verify PVC resize feature
+    """
+    pvc_size = 10
+    pvc_size_new = 40
+    patch_param = f'{{"spec": {{"resources": {{"requests": {{"storage": "{pvc_size_new}Gi"}}}}}}}}'
+
+    sc_obj = storageclass_factory(interface=interface)
+
+    # Create PVC and pod
+    log.info("Creating PVC and pod.")
+    pvc_obj = pvc_factory(interface=interface, storageclass=sc_obj, size=pvc_size)
+    pod_obj = pod_factory(interface=interface, pvc=pvc_obj)
```

Comment:
suggest moving this to the test setup to keep the test/setup separation logic.
For that, you can create a fixture in line 10 as follows:
```python
    @pytest.fixture(autouse=True)
    def test_setup(self, interface, storageclass_factory, pvc_factory, pod_factory):
        """
        Create resources for the test

        """
        self.sc_obj = storageclass_factory(interface=interface)
        self.pvc_obj = pvc_factory(interface=interface, storageclass=sc_obj, size=pvc_size)
        self.pod_obj = pod_factory(interface=interface, pvc=pvc_obj)

```

And then, change the test function signature to:
```python
def test_pvc_resize(test_setup):
```

Reply:
Done.